cpp/example_code/cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp

```diff
@@ -346,10 +346,11 @@ bool AwsDoc::TopicsAndQueues::messagingWithTopicsAndQueues(
                         Aws::SQS::Model::QueueAttributeName::QueueArn);
                 if (iter != attributes.end()) {
                     queueARN = iter->second;
-                    std::cout << "The queue ARN '" << queueARN << "' has been retrieved."
+                    std::cout << "The queue ARN '" << queueARN
+                              << "' has been retrieved."
                               << std::endl;
                 }
-               // snippet-end:[cpp.example_code.cross-service.topics_and_queues.GetQueueAttributes1]
+                    // snippet-end:[cpp.example_code.cross-service.topics_and_queues.GetQueueAttributes1]
```

Comment:
Is the indentation here intentional?

Reply:
it doesn't matter and the automatic formatter did it.